content/2019-munich/register.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Register
+---
+
+## Register to attend
+
+Attendee registration is now open!
+
+<a class="btn btn-lg btn-default" href="http://www.cvent.com/d/cyqq4q/4W" target="_blank" role="button">
```

Comment:
Just want to make sure we only sell limited waves of tickets at first (like maybe 100 at first)? Until we know how many exactly we need for staff, diversity, sponsors, etc.

Reply:
Yes, we can cap it at first 100

Reply:
yes, we will cap to the first 120